README.rst

```diff
@@ -1205,6 +1205,8 @@ Changelog
 
 * v0.10.1     - Not released yet
 
+  * Unpinn python-blosc (#51)
```

Comment:
Unpin can be used as a verb if you want, but I assume that you would want to use past tense in release notes/change log: `unpinned`.